src/components/WelcomeModal/CarouselCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { useRef, useEffect } from "react";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { Body } from "@leafygreen-ui/typography";
+import { size } from "constants/tokens";
+
+type CarouselCardProps = {
+  img?: string;
+  movie?: string;
+  subtitle: string;
+  description: string;
+  visible: boolean;
+};
+
+const CarouselCard: React.VFC<CarouselCardProps> = ({
+  img,
+  movie,
+  subtitle,
+  description,
+  visible,
+}) => {
+  const videoRef = useRef<HTMLVideoElement>(null);
+  useEffect(() => {
+    if (visible && videoRef.current) {
+      videoRef.current.play();
+    }
+  }, [visible]);
+  return (
+    <CardContainer>
+      <Body weight="medium">{subtitle}</Body>
+      <Body>{description}</Body>
+      {img && <ImgContainer src={`/static/img/${img}`} />}
+      {movie && (
+        <MovieContainer
+          loop
+          controls
+          src={`/static/img/${movie}`}
+          ref={videoRef}
+        />
+      )}
+    </CardContainer>
+  );
+};
+
+const containerStyles = css`
+  height: 250px;
```

Comment:
Could we not use a fixed height? For videos, it adds some auto margins on the side and makes them even smaller. Photos are having their aspect ratio changed too which makes them look odd and smooshed.